games-emulation/retroarch/retroarch_assets-20180129.recipe

```diff
@@ -0,0 +1,35 @@
+SUMMARY="Media assets for RetroArch"
+DESCRIPTION="RetroArch is a frontend for emulators, game engines and media \
+players. This package provides assets such as icons for the GUI."
+HOMEPAGE="https://libretro.com/"
+COPYRIGHT="2010-2018 The RetroArch Team"
+LICENSE="CC-BY-4.0"
+REVISION="1"
+srcGitRev="505ad6205e5fd8a155daad02ab39953a0b15b0a9"
+SOURCE_URI="https://github.com/libretro/retroarch-assets/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="de37772b00b755b80aafe5d4df0713c0fe6b0e5a79017d94e43f279e798aa85d"
+SOURCE_FILENAME="retroarch_assets-$portVersion.tar.gz"
+SOURCE_DIR="retroarch-assets-$srcGitRev"
+
+ARCHITECTURES="any"
+
+PROVIDES="
+	retroarch_assets = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+BUILD()
+{
+	rm -rf "$retroarchDataDir"/Makefile \
+		"$retroarchDataDir"/configure \
+		"$retroarchDataDir"/COPYING \
+		"$retroarchDataDir"/src
+}
+
+INSTALL()
+{
+	retroarchDataDir="$dataDir/retroarch/assets/"
+	mkdir -p "$retroarchDataDir"
+	cp -ar * "$retroarchDataDir"
```

Comment:
Hmm, I think my idea of adding a `BUILD()` to remove some files and the src tree was not bright :)
That's because removing source files in the build directory might have side effect on the `--extract-patchset` feature of haikuporter.

So, since you made a small mistake in your current `BUILD()` (`retroarchDataDir` is not defined, so the rm would not delete any files), it gives me a chance to suggest another variant:
```diff
-BUILD()
-{
-	rm -rf "$retroarchDataDir"/Makefile \
-		"$retroarchDataDir"/configure \
-		"$retroarchDataDir"/COPYING \
-		"$retroarchDataDir"/src
-}
-
 INSTALL()
 {
-	retroarchDataDir="$dataDir/retroarch/assets/"
+	retroarchDataDir="$dataDir"/retroarch/assets
 	mkdir -p "$retroarchDataDir"
-	cp -ar * "$retroarchDataDir"
+	# copy everything but configure, COPYING, Makefile and src
+	cp -ar [abd-rt-z]* "$retroarchDataDir"
+	rm -f "$retroarchDataDir"/xmb/convert.sh
 }
```

Reply:
My only issue with this variant is that it will need to be updated if there are extra folders added by upstream in the media/assets directory that start with letters c or s.

Reply:
You are right. Your first `INSTALL()` is better. Even if it copies src and then removes it, that's still better than having unneeded patterns.

Regarding the `xmb/convert.sh` shell script, which converts some svg to png, we don't have inkscape but we might, some day, find some way to use [media-libs/libsvgtiny](https://github.com/haikuports/haikuports/tree/master/media-libs/libsvgtiny) for doing the conversion.
I guess this conversion should be done at build time.
Well, I guess retroarch does complain if it only finds the svg.

Reply:
I will revert the changes to INSTALL()
Regarding convert.sh it's not necessary to retroarch. It's a convenience script for icon designers, that is not needed by the end user. Same goes with NPMApng2PMApng.py and I will have them removed as well.